src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts

```diff
@@ -1443,4 +1471,92 @@ export const reasonableAdjustment = (
     title: sectionTitles['reasonAbleAdjustment'],
     rows: getSectionSummaryList(SummaryData, content),
   };
-};
\ No newline at end of file
+};
+
+export function genderChose(choice: string, language: string|undefined): string {
```

Comment:
Can you please remove all below individual functions and put all these translations in content CY translations as separate keys as how I did for recent Yes No translation changes.